workflows/spaceranger/spaceranger_count.wdl

```diff
@@ -232,8 +232,15 @@ task run_spaceranger_count {
             return darkimages
 
         has_cyta = not_null('~{cytaimage}')
+        probe_set = "~{probe_set}"
+        probe_set = "" if probe_set is None else probe_set
         if not_null('~{probe_file}'):
-            call_args.extend(['--probe-set=~{probe_file}', '--filter-probes=~{filter_probes}'])
+            call_args.append('--probe-set=~{probe_file}')
+            if version.parse('~{spaceranger_version}') >= version.parse('2.0.0'):
+                call_args.append('--filter-probes=~{filter_probes}')
+            else:
```

Comment:
Please remove `else` branch. This is to be consistent with what we already set for `spaceranger_count.wdl` now.

Reply:
Not sure I understand, if I remove the else clause then it would not work with 1.3.1, meaning that probes will always be filtered. Is that the intended action?